test/10-main.spec.js

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  base64UrlFromPdfBytes,
+  canonicalize,
+  hashCanonicalized
+} from '../lib/canonize.js';
```

Comment:
If testing the main module API, use the normal entry point.  `index.js` wasn't updated after rename, so the tests pass, but the module is broken.  If testing some part of the lib on it's own, or private local APIs, then use whatever is needed.
```suggestion
} from '../lib/index.js';
```